### ISSUE TITLE
models used for submission to the competition

### ISSUE THREAD

**Issue by Willy97r** [yorkeyao/DataCV2024#2]
Thanks for oraganizing this interesting competetion. Going through your paper and the competition requirements, I have this question -

You require to submit the detection results on test-A and test-B. But how do you ensure that indeed an 8000 sampled dataset was used for training a detection model and that a retinanet model was used for training with the parameters specified in the baseline code? How can all of these information be inferred from the detection submission file to ensure fairness?

**Comment by yorkeyao** [comment]
Thank you for your question. For participants that would like to be in the final ranking. We will ask them to release the code publicly. We will run their code to ensure it matches the result on the evaluation server.

**Comment by Willy97r** [comment]
Thank you for the clarification and it makes sense. Just to confirm- the participants should not be tuning retinanet for this? As in use the same epochs given in the config file (12) with same learning rate and other params in the retinanent config file?

**Comment by yorkeyao** [comment]
Yes for challenges we do not allow participants to tune models.

**Comment by Willy97r** [comment]
Thank you. Is it allowed to change samples_per_gpu /workers_per_gpu in the custom_tss.py for the retiannet configuration to fully utilize the capability of my current gpu? Currently its only set to 2 . Thanks.

**Comment by yorkeyao** [comment]
Yes, samples_per_gpu /workers_per_gpu are allowed to change
